weblate/addons/properties.py

```diff
@@ -113,12 +116,12 @@ def filter_lines(lines):
     return result
 
 
-def format_file(filename) -> None:
+def format_file(filename, case_sensitive) -> None:
```

Comment:
Please add type annotations when changing the code:

```suggestion
def format_file(filename: str, case_sensitive: bool) -> None:
```

We're not yet at a state where the whole Weblate would be type-checkable, but we want to get there.

Reply:
Suggested change applied. I will take note of putting type hints to parameters.